packages/shared/src/net/retry.ts

```diff
@@ -21,6 +22,7 @@ const defaultOptions: ExecuteWithHttpRetriesOptions = {
     "ECONNREFUSED", // (Connection refused): No connection could be made because the target machine actively refused it. This usually results from trying to connect to a service that is inactive on the foreign host.
     "ECONNRESET", //  (Connection reset by peer): A connection was forcibly closed by a peer. This normally results from a loss of the connection on the remote socket due to a timeout or reboot. Commonly encountered via the http and net modules.
   ],
+  httpStatusCodesToRetry: [429], // 429 Too Many Requests
```

Comment:
Let's update the TSDoc to include the default http status to retry?

Reply:
Also, since we're retrying on 429 by default (good thing!), I think we should prob increase the initial delay on `executeWithNetworkRetries` from the one defined on `executeWithRetries` (10ms) to something like 100ms, otherwise we'll probably use all the attempts to retry too close one to another and not get away from "too many requests".

Thoughts?

Reply:
I agree, but would probably make it a 1000 to be safe